pkg/kor/clusterroles.go

```diff
@@ -191,18 +191,18 @@ func GetUnusedClusterRoles(filterOpts *filters.Options, clientset kubernetes.Int
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "Failed to process cluster role : %v\n", err)
 	}
+	if opts.DeleteFlag {
+		if diff, err = DeleteResource2(diff, clientset, "", "ClusterRole", opts.NoInteractive); err != nil {
```

Comment:
Not directly related to this PR, but can we rename DeleteResource2 to DeleteResource and remove the old DeleteResource